contracts/QuestFactory.sol

```diff
@@ -115,45 +128,61 @@ contract QuestFactory is Initializable, OwnableUpgradeable, AccessControlUpgrade
         return address(0);
     }
 
+    /// @dev grant the create quest role to an account
+    /// @param account_ The account to grant the create quest role to
     function grantCreateQuestRole(address account_) public {
         _grantRole(CREATE_QUEST_ROLE, account_);
     }
 
+    /// @dev revoke the create quest role from an account
+    /// @param account_ The account to revoke the create quest role from
     function revokeCreateQuestRole(address account_) public {
         _revokeRole(CREATE_QUEST_ROLE, account_);
     }
 
+    /// @dev grant the default admin role and the create quest role to the owner
     function grantDefaultAdminAndCreateQuestRole() public onlyOwner {
         _grantRole(DEFAULT_ADMIN_ROLE, msg.sender);
         _grantRole(CREATE_QUEST_ROLE, msg.sender);
     }
 
+    /// @dev set the claim signer address
```

Comment:
Should we also add `@param` for address? (same with functions below)

Reply:
yes, thanks added.